### DISCUSSION TITLE
Can We Have Substitute Preserve Case?

### DISCUSSION THREAD

**Discussion by tonny-sofijan** [neovim/neovim#31732] [Ideas]
Like the title, could we have **Substitute Preserve Case** feature like in VSCode and phpstorm? I think it's quite crucial and will help immensely for coding. We can use the idea from abolish.vim. 
Thanks.

**Comment by zeertzjq** [comment]
Duplicate of #8192

**Comment by tonny-sofijan** [comment]
Noted, thanks.
